src/assets/translations/ko/main.json

```diff
@@ -763,25 +859,25 @@
     "csv": {
       "fileName": "ShapeShift Transaction History",
       "txid": "TX ID",
-      "type": "TX 유형",
-      "timestamp": "TX Timestamp",
-      "status": "TX 상태",
-      "minerFee": "마이너 수수료",
-      "minerFeeCurrency": "마이너 수수료 유형",
-      "inputAmount": "입금액",
-      "inputCurrency": "입금 자산",
-      "inputAddress": "입금 주소",
-      "outputAmount": "출금액",
-      "outputCurrency": "출금 자산",
-      "outputAddress": "출금 주소"
+      "type": "TX type",
+      "timestamp": "Timestamp",
+      "status": "Status",
+      "minerFee": "Miner fee",
+      "minerFeeCurrency": "Miner fee Currency",
+      "inputAmount": "Input amount",
+      "inputCurrency": "Input currency",
+      "inputAddress": "Input address",
+      "outputAmount": "Output amount",
+      "outputCurrency": "Output currency",
+      "outputAddress": "Output address"
```

Comment:
i mean i don't speak korean but this looks like english to me

Reply:
It was intentional. The current CSV plugin isn't parsing unicode properly; seeing pretty much all non-Latin based character systems with broken text. It was something @XrosFade actually found, I meant to bring it up eventually but figured you guys had bigger tasks at hand ;) Should've mentioned it in the request, my bad!